pkg/logger/logger.go

```diff
@@ -117,6 +117,8 @@ func NewLogger(logFormat, logLevel string) (*ZapLogger, error) {
 
 	cfg := zap.NewProductionConfig()
 	cfg.Level = zap.NewAtomicLevelAt(level)
+	cfg.EncoderConfig.TimeKey = "timestamp"
```

Comment:
Given consideration to possible side effects of this internally?

Reply:
Good point. I think this has to be reverted. Thanks!

Reply:
Actually, it is okay.